packages/gatsby/package.json

```diff
@@ -18,6 +18,7 @@
     "@babel/runtime": "^7.15.4",
     "@babel/traverse": "^7.15.4",
     "@babel/types": "^7.15.4",
+    "@builder.io/partytown": "^0.5.2",
```

Comment:
I don't like that we have same dep in here and in `gatsby-script`. With us using `^` version selectors generally this might lead to different version of `@builder.io/partytown` installed for gatsby and for gatsby-script and potentially cause problems.

If we are not putting script stuff in `gatsby` package directly, I would want to propose that instead of adding this dependency to `gatsby`, I would expose functionality we need in core from `gatsby-script`. From what I can tell that's `copyLibFiles`.

Other option (which I start to like even better), is just make `gatsby-script` a plugin (so include the `gatsby-node` / `gatsby-ssr` pieces there instead of internal plugin in gatsby). This would allow dropping this dep from gatsby as everything is in `gatsby-script` (apart from re-export from gatsby and reverse proxies)